exclusion_zones.civmap.json

```diff
@@ -92,7 +92,7 @@
 		{
 			"name": "Fort Sampaloc",
 			"owner": "Lambat",
-			"contact": "Lambat",
+			"contact": "Lambat (https://discord.gg/EYwjerUU3V)",
```

Comment:
I'm worried the closing parenthesis will become part of the url, can you remove the parentheses please?